pytorch/buildspec.yml

```diff
@@ -1,8 +1,8 @@
 account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
 region: &REGION <set-$REGION-in-environment>
 framework: &FRAMEWORK pytorch
-version: &VERSION 1.12.1
-short_version: &SHORT_VERSION "1.12"
+version: &VERSION 1.11.0
+short_version: &SHORT_VERSION "1.11"
```

Comment:
Please ensure that the `pytorch/buildspec-1-11.yml` file is usable to build these images, before this PR is merged, so that we can ensure the config in the buildspec file used by the  pipeline is correct.

Reply:
updated